charts/trusted-attestation-controller/README.md

```diff
@@ -0,0 +1,134 @@
+# Trusted Attestation Controller Helm Chart
+
+Trusted Attestation Controller (TAC) is a K8s controller for reconciling the [QuoteAttestation](https://github.com/intel/trusted-certificate-issuer/blob/main/api/v1alpha1/quoteattestation_types.go) requests initiated by the [Trusted Certificate Service (TCS)](https://github.com/intel/trusted-certificate-issuer). It is a proxy between the TCS and the key server(s) which supports attestation services. The key servers could plugin to the controller by implementing the API provided by the controller.
+
+To learn more check the documentation [here](https://github.com/intel/trusted-attestation-controller).
+
+## Prerequisites
+
+Prerequisites for building and running Trusted Attestation Controller:
+
+- Kubernetes cluster with running [Trusted Certificate Service](https://github.com/intel/trusted-certificate-issuer)
+- Either KMRA or a KMIP-compliant key server exposed via iSecL's Key Broker Service
+- git, or similar tool, to obtain the source code
+- Docker, or similar tool, to build container images
+- Container registry ([local](https://docs.docker.com/registry/deploying/) or remote)
```

Comment:
There are prerequisites only if one wants to build the TAC from the source, which could be removed from here.